workloads/kube-burner/workloads/large-networkpolicy-egress/README.md

```diff
@@ -0,0 +1,30 @@
+
+## large-networkpolicy-egress
+
+With the help of [large-networkpolicy-egress] customer cases that combined with node-density-heavy, network policy and egress firewall
+
+## How to run large-networkpolicy-egress tests?
+
+The environmental variables and steps to kick off this test can be found [here](https://github.com/cloud-bulldozer/e2e-benchmarking/blob/master/workloads/kube-burner/README.md#kube-burner-e2e-benchmarks)
+
+## What are the test cases that this workload can currently run?
+
+### Case for large namespaces
+
+- 2500 namespaces, ITERATION=2500, one namespace each ITERASTION, great than 50 worker node
+- 30 pods in each namespace. One server pod, one client pod, one egress testing pod. specify POD_RPLICAS=10 for each type pod, default value of POD_RPLICAS is 10
```

Comment:
This is not quite clear to me. Do you mean 'POD_RPLICAS number of server pods, POD_RPLICAS number of client pods and POD_RPLICAS number of egress testing pods. Default POD_RPLICAS is 30'?